crates/graphql/src/schema.rs

```diff
@@ -84,7 +85,7 @@ impl Creator {
                      trait_type, value, ..
                  }| {
                     *groups
-                        .entry(trait_type.unwrap().to_lowercase())
+                        .entry(trait_type.unwrap().to_lowercase().clone())
```

Comment:
```suggestion
                        .entry(trait_type.unwrap().to_lowercase())
```
Accidentally reverted a fixup from dev